routes/personal/personal.spec.js

```diff
@@ -65,6 +66,29 @@ describe('Test /personal responses', () => {
     })
   })
 
+  describe('Test /personal/residence responses', () => {
+    test('it returns a 422 with no option selected', async () => {
+      const response = await request(app)
+        .post('/personal/residence')
+        .send({ redirect: '/personal/address' })
+      expect(response.statusCode).toBe(422)
+    })
+
+    test('it returns a 302 when selecting NO', async () => {
+      const response = await request(app)
+        .post('/personal/residence')
+        .send({ redirect: '/', residence: 'No' })
+      expect(response.statusCode).toBe(302)
+    })
+
+    test('it returns a 302 when selecting YES', async () => {
```

Comment:
You _could_ write one more test for some bad value (ie, not 'Yes', or 'No'), but I already wrote a bunch somewhere else and it would just be boilerplate.

So, on second thought, this is fine.